common/i2c_reg.h

```diff
@@ -15,7 +15,7 @@
 
 // This array helps us simplify the use of different I2C devices in the board.
 extern uint32_t I2C_BASE[];
-
+// the following comments might be outdated. please refer to an updated wiki page
```

Comment:
rather than add a comment that says that the comment is bad just either remove the comment or write that it's for Rev1

Reply:
Okay. Will fix that now.